src/ImageSharp/Formats/Webp/Lossy/WebpLossyDecoder.cs

```diff
@@ -252,18 +270,35 @@ private void ParseIntraMode(Vp8Decoder dec, int mbX)
                     }
 
                     top.CopyTo(modes);
-                    modes = modes.Slice(4);
+                    modes = modes[4..];
                     left[y] = (byte)yMode;
                 }
             }
 
             // Hardcoded UVMode decision tree.
-            block.UvMode = (byte)(this.bitReader.GetBit(142) == 0 ? 0 :
-                           this.bitReader.GetBit(114) == 0 ? 2 :
-                           this.bitReader.GetBit(183) != 0 ? 1 : 3);
+            if (this.bitReader.GetBit(142) == 0)
+            {
+                // Hardcoded UVMode decision tree.
```

Comment:
Do we really need this comment on every assignment?

Reply:
I think so. I think the reference source is not hard coded and there’s scope for potential enhancement here